aleks/resume.md

```diff
@@ -1,29 +1,46 @@
-## Résumé: Aleksandar Todorović 	
+# Résumé: Aleksandar Todorović 	
 
 ### Personal Details:
 
 #### Affiliation:
-Bits of Freedom, Open Web Fellow
+[Bits of Freedom](https://www.bof.nl/), Open Web Fellow
 
 #### Contact:
-* **Email**: [aleksandar@mozillafoundation.org](mailto:aleksandar@mozillafoundation.org)
-* **Twitter**: [@r3bl_](https://twitter.com/r3bl_)
-* **GitHub**: [@aleksandar-todorovic](https://github.com/aleksandar-todorovic)
 
-As part of the 2017 Open Web Fellows program, I completed/participated in the following:
+* **Emails**: [aleksandar@mozillafoundation.org](mailto:aleksandar@mozillafoundation.org) / [aleksandar@r3bl.me](mailto:aleksandar@r3bl.me) / [aleksandar.todorovic@bof.nl](mailto:aleksandar.todorovic@bof.nl)
+* **Microblogging**: [@r3bl_](https://twitter.com/r3bl_) on Twitter, [@r3bl@mastodon.social](https://mastodon.social/@r3bl) on ActivityPub/OStatus fediverse. 
+
+#### Other useful links:
+
+* **Personal website:** [r3bl.me](https://r3bl.me)
+* **Personal blog:** [blog.r3bl.me](https://blog.r3bl.me)
+* **GitHub**: [@aleksandar-todorovic](https://github.com/aleksandar-todorovic) for smaller, personal projects / [@r3bl](https://github.com/r3bl) for publicly available web projects
+
+As part of the 2017 Open Web Fellows program, I completed/participated/worked on the following:
 
 #### EVENTS & TALKS
 
 Event | Location | Date | Role
 ----- | -------- | ---- | -----
-[Cool Event](URL) | Some place, Rad country | Sept 2017 | Speaker  
-
+Mozilla Festival | London, UK | 
 
 #### PROJECTS
 TITLE | TECH USED | LINK | DESCRIPTION
 ----- | --------- | ---- | ------------
+| My Data Done Right | AngularJS, Django, WordPress | [mydatadoneright.eu](https://mydatadoneright.eu) | A web portal that will allow EU citizens to easily exercise their rights as the data subjects under the GDPR. Main project for the fellowship. |
+| Bootstraped Timeline | Bash, Python, AngularJS, HTML5/CSS3 | N/A (writeup to be published soon) | In a world where "black box" algorithms dictate our attention, what happens when someone decides to take a completely different approach? |
+| We're not looking at the same webpage | HTML5/CSS3, JavaScript, JQuery | N/A (to be released soon) | An art project in a form of a webpage explaining A/B testing in a way that the webpage itself is unique to every visitor. |
```

Comment:
Cool! Not sure this is what you mean but I like the way wooorm does his website : http://wooorm.com/
> P.S. I made this site for you, `0b11b0`, try a `different design`

Reply:
Not quite like that. You'll receive more info about it in an email. It will be released in a month from now.

Reply:
Whoot, please do invite me to the webring/beta test group.